packages/inventory-general-info/src/dataMapper.js

```diff
@@ -9,15 +9,32 @@ import {
     OutlinedArrowAltCircleDownIcon
 } from '@patternfly/react-icons';
 import { sortable } from '@patternfly/react-table';
+import { Tooltip } from '@patternfly/react-core';
 
 export const statusHelper = {
-    UP: <OutlinedArrowAltCircleUpIcon className="ins-c-inventory__detail--up" />,
-    DOWN: <OutlinedArrowAltCircleDownIcon className="ins-c-inventory__detail--down" />
+    UP: <Tooltip content={<div>
```

Comment:
Why all these divs in Tooltip?

According to docs, a string should OK.

Reply:
Good poin! I've removed all divs and spans in here.